.nvmrc

```diff
@@ -1 +1 @@
-8.7.0
+8.8.0
```

Comment:
We should be using latest to test the app but minimal version should stay at 8.0.0